x-pack/plugins/infra/public/alerting/inventory/rule_data_formatters.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import {
+  ALERT_EVALUATION_THRESHOLD,
+  ALERT_EVALUATION_VALUE,
+  ALERT_START,
+} from '@kbn/rule-data-utils';
+import { modifyUrl } from '@kbn/std';
+import { ObservabilityRuleTypeFormatter } from '../../../../observability/public';
+
+export const formatReason: ObservabilityRuleTypeFormatter = ({ fields }) => {
+  const actualCount = fields[ALERT_EVALUATION_VALUE];
+  const thresholdCount = fields[ALERT_EVALUATION_THRESHOLD];
+  const reason = i18n.translate('xpack.infra.metrics.alerting.inventory.alertReasonDescription', {
+    defaultMessage: 'Current value is {actualCount} (threshold of {thresholdCount}).',
+    values: {
+      actualCount,
+      thresholdCount,
+    },
+  });
+
+  const alertStartDate = fields[ALERT_START];
+  const timestamp = alertStartDate != null ? new Date(alertStartDate).valueOf() : null;
+  const link = modifyUrl('/app/metrics/link-to/default/metrics', ({ query, ...otherUrlParts }) => ({
```

Comment:
Does that URL actually exist? I can only find `metrics/link-to/{node-type}-detail/{nodeId}` and `metrics/link-to/host-detail-via-ip/{hostIp}`. And those don't take a `time` query parameter.

Reply:
I see. In that case I will need the serialized params I guess to store the node_type (host, pod, container etc), so that I can construct the url dynamically. 

Also since these don't take a `time` query parameter, then I don't need to use the modifyUrl, right? Do we have another function I can use that constructs the url prepending the base path?

Reply:
I'll use the link to default metrics page as discussed